### DISCUSSION TITLE
Comment toggle stopped working after update (shows up in cheatsheet)

### DISCUSSION THREAD

**Discussion by webel** [NvChad/NvChad#2889] [Q&A] [answered]
I've updated to NvChad 2.5 (very cool thank you thank you) but now the toggle comments command isn't working, although the  hint list thing does show up with the leader + / hint showing up first. 
What to do?

**Comment by siduck** [comment, marked as answer]
update nvim to 0.10

**Comment by bryanmaina** [comment]
It doesn't work for me under 

![image](https://github.com/NvChad/NvChad/assets/8012851/9b83c99d-03c3-46d6-8da8-23f0a7cdba62)

**Reply by siduck** [reply]
@bryanmaina  try the v0.10 stable release, either from bob neovim version manager or neovim releases page
